files/jcl/ZWESECUR.jcl

```diff
@@ -520,32 +557,36 @@ $$
 
 /* DEFINE ZOWE SERVER PERMISIONS ................................... */
 
-/* required updates:                                                 */
-/* - change "fac_owning_acid" to the acid that owns IBMFAC           */
-
-/* permit Zowe main server to use XMEM, cross memory server          */
-  TSS ADD(fac_owning_acid) IBMFAC(ZWES.IS)
+/* permit Zowe main server to use XMEM cross memory server           */
+  TSS ADD(&FACACID.) IBMFAC(ZWES.IS)
   TSS WHOHAS IBMFAC(ZWES.IS)
   TSS PERMIT(&ZOWEUSER.) IBMFAC(ZWES.IS) ACCESS(READ)
 
+/* permit Zowe main server to create a user's security environment   */
+/* ATTENTION: Defining the BPX.DAEMON or BPX.SERVER profile makes    */
+/*            z/OS UNIX switch to z/OS UNIX level security. This is  */
+/*            more secure, but it can impact operation of existing   */
+/*            applications. Test this thoroughly before activating   */
+/*            it on a production system.                             */
+  TSS ADD(&FACACID.) IBMFAC(BPX.)
+  TSS WHOHAS IBMFAC(BPX.SERVER)
+  TSS PER(&ZOWEUSER.) IBMFAC(BPX.SERVER) ACC(UPDATE)
```

Comment:
The same note as in RACF section: &ZOWEUSER should also have UPDATE access to BPX.DAEMON.
`TSS PER(&ZOWEUSER.) IBMFAC(BPX.DAEMON) ACC(UPDATE)`

Reply:
according to the error message, ZOWEUSEr only needs BPX.SERVER, which makes sense, IDz also only needs BPX.SERVER. I always found it weird that ZSS needs both, but they must have a reason.